Combobox.js

```diff
@@ -930,6 +937,9 @@ define([
 				var promise = sup.apply(this, arguments);
 
 				return promise.then(function () {
+					this.setAttribute("aria-owns", this.dropDown.id);
+					this.inputNode.setAttribute("aria-controls", this.dropDown.id);
```

Comment:
I think the `aria-owns` is now being handled by HasDropDown so all you need is to set `aria-controls`.